proj-android/PowerAuthLibrary/src/main/java/io/getlime/security/powerauth/biometry/impl/BiometricHelper.java

```diff
@@ -134,4 +136,78 @@
             return null;
         }
     }
+
+    /**
+     * Determine if the current device should explicitly fallback {@code FingerprintManager} based
+     * authentication.
```

Comment:
I would improve the comment here to explain what is the idea - it took me some time to build a Venn diagram in my head for the conditions below. Also, I would look if we could write the conditions easier (specifically, I was struggling with the negative comparison for Android P, and figuring out what the condition means).

Reply:
I second to that, the comparison with the P looks out-of-place without explanation.

Reply:
OK, I explained that in more detail in my last commit.